src-db/database/model/functions/C_INVOICE_UPDATE_TAXES.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0"?>
+  <database name="FUNCTION C_INVOICE_UPDATE_TAXES">
+    <function name="C_INVOICE_UPDATE_TAXES" type="NULL">
+      <parameter name="p_ep_instance" type="VARCHAR" mode="in">
+        <default/>
+      </parameter>
+      <body><![CDATA[/*************************************************************************
+* The contents of this file are subject to the Openbravo  Public  License
+* Version  1.0  (the  "License"),  being   the  Mozilla   Public  License
+* Version 1.1  with a permitted attribution clause; you may not  use this
+* file except in compliance with the License. You  may  obtain  a copy of
+* the License at http://www.openbravo.com/legal/license.html
+* Software distributed under the License  is  distributed  on  an "AS IS"
+* basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the
+* License for the specific  language  governing  rights  and  limitations
+* under the License.
+* The Original Code is Openbravo ERP.
+* The Initial Developer of the Original Code is Openbravo SLU
+* All portions are Copyright (C) 2014 Openbravo SLU
+* All Rights Reserved.
+* Contributor(s):  ______________________________________.
+************************************************************************/
+
+p_record_id C_INVOICE.C_Invoice_ID%TYPE;
+p_docAction C_INVOICE.DocAction%TYPE;
+
+TYPE RECORD IS REF CURSOR;
+Cur_Params RECORD;
+
+BEGIN
+
+  FOR Cur_Params IN (
+    SELECT *
+    FROM ad_ep_instance_para
+    WHERE ad_ep_instance_id = p_ep_instance
+    ) LOOP
+    IF (cur_params.parametername LIKE 'DocAction') THEN
+      p_docaction := Cur_Params.p_string;
+    ELSIF (cur_params.parametername LIKE 'Record_ID') THEN
+      p_record_id := cur_params.p_string;
+    END IF;
+  END LOOP;
+
+  IF (p_docAction IN ('CO')) THEN
```

Comment:
check this.

Reply:
a que te referis? al IN?

Reply:
Si y que ademas te asegures lo que te dije al anular una factura cuando no fue entregada (sin albaran)

Reply:
resuelto